includes/functions/content-sync.php

```diff
@@ -311,3 +315,60 @@ function get_post_data( $post ) {
 	 */
 	return apply_filters( 'sophi_post_data', $data );
 }
+
+/**
+ * Check if the current data has been already tracked recently and should be skipped.
+ *
+ * @since 1.3.0
+ *
+ * @param array $data Data being tracked
+ *
+ * @return boolean
+ */
+function maybe_skip_track_event( $data ) {
+	// Ignore modifiedAt.
+	unset( $data['modifiedAt'] );
+
+	$transient_hash = substr( wp_hash( wp_json_encode( $data ) ), 0, 8 );
```

Comment:
```suggestion
	$transient_hash = substr( md5( wp_json_encode( $data ) ), 0, 8 );
```

As this isn't used for cryptographic purposes, you can use an md5 instead of `wp_salt()`. By default the latter uses the AUTH salt which is only intended to be used for authentication purposes.

Transient names can be up to 172 characters long, so I don't think you need to truncate the hash but it's no big deal.